aliases/git/nu_alias_git.nu

```diff
@@ -4,200 +4,190 @@ def git_current_branch [] {
 
 def git_main_branch [] {
     git remote show origin
-    | lines
-    | str trim
-    | find --regex 'HEAD .*?[：: ].+'
-    | first
-    | str replace 'HEAD .*?[：: ](.+)' '$1'
+        | lines
+        | str trim
+        | find --regex 'HEAD .*?[：: ].+'
+        | first
+        | str replace 'HEAD .*?[：: ](.+)' '$1'
 }
 
 #
 # Aliases
 # (sorted alphabetically)
 #
 
-export alias ga                    = git add
-export alias gaa                   = git add --all
-export alias gapa                  = git add --patch
-export alias gau                   = git add --update
-export alias gav                   = git add --verbose
-export alias gap                   = git apply
-export alias gapt                  = git apply --3way
-
-export alias gb                    = git branch
-export alias gba                   = git branch -a
-export alias gbd                   = git branch -d
-export alias gbD                   = git branch -D
-export alias gbl                   = git blame -b -w
-export alias gbnm                  = git branch --no-merged
-export alias gbr                   = git branch --remote
-export alias gbs                   = git bisect
-export alias gbsb                  = git bisect bad
-export alias gbsg                  = git bisect good
-export alias gbsr                  = git bisect reset
-export alias gbss                  = git bisect start
-
-export alias gc                    = git commit -v
-export alias gc!                   = git commit -v --amend
-export alias gcn                   = git commit -v --no-edit
-export alias gcn!                  = git commit -v --no-edit --amend
-export alias gca                   = git commit -v -a
-export alias gca!                  = git commit -v -a --amend
-export alias gcan!                 = git commit -v -a --no-edit --amend
-export alias gcans!                = git commit -v -a -s --no-edit --amend
-export alias gcam                  = git commit -a -m
-export alias gcsm                  = git commit -s -m
-export alias gcas                  = git commit -a -s
-export alias gcasm                 = git commit -a -s -m
-export alias gcb                   = git checkout -b
-export alias gcd                   = git checkout develop
-export alias gcf                   = git config --list
-
-export alias gcl                   = git clone --recurse-submodules
-export alias gclean                = git clean -id
-
-export def gpristine [] {
-    git reset --hard
-    git clean -dffx
-}
```

Comment:
this has been added in #485 and cannot be replaced with
```
export alias gpristine = ((git reset --hard); (git clean -dffx))
```

see
```
> export alias gpristine = ((git reset --hard); (git clean -dffx))
Error: nu::parser::cant_alias_expression

  × Can't create alias to expression.
   ╭─[entry #12:1:1]
 1 │ export alias gpristine = ((git reset --hard); (git clean -dffx))
   ·                          ───────────────────┬───────────────────
   ·                                             ╰── aliasing FullCellPath is not supported
   ╰────
  help: Only command calls can be aliased.
```

Reply:
I've moved commands in parentheses into `def`s. Is it the best fix here?

Reply:
you can also just define the command directly
```
def gpristine [] { ... }
```
instead of 
```
def gpristine-command [] { ... }
export alias gpristine = gpristine-command
```
:+1:

Reply:
Done. :)

Reply:
:partying_face: